biz.aQute.bndlib/src/aQute/bnd/osgi/Jar.java

```diff
@@ -1246,4 +1247,15 @@ public void removeSubDirs(String dir) {
 			.keySet());
 		subDirs.forEach(subDir -> removePrefix(subDir + "/"));
 	}
+
+	private static final Predicate<String> pomXmlFilter = new PathSet("META-INF/maven/*/*/pom.xml").matches();
+
+	public Stream<Resource> getPomResources() {
```

Comment:
Since this method is now API on Jar, perhaps a more precise method name is `getPomXmlResources`? This is because a jar may also have pom properties resources and the name `getPomResources` is a bit ambiguous in that regard.